### ISSUE TITLE
Saving with sublime and this plugin is extremely slow

### ISSUE THREAD

**Issue by raurir** [sveltejs/prettier-plugin-svelte#301]
As demonstrated here: https://github.com/raurir/svelte-code-linting#autoformatting

Similar if not identical to #60

**Comment by dummdidumm** [comment]
Is it also slow with prettier formatting large JS/TS files?

**Comment by raurir** [comment]
@dummdidumm No, it's almost instant, for example [this file](https://github.com/sveltejs/prettier-plugin-svelte/blob/master/src/print/index.ts) formats in <0.3 seconds.

**Comment by probablykasper** [comment]
This is how long formatting takes for me for my codebase of 207 files. Svelte files are 10x slower than Typescript:

Type | File count | Time | Time per file
-- | --: | --: | --: |
.css | 2 | 480 | 240
.html | 1 | 79 | 79
.js | 2 | 36 | 18
.json | 2 | 19 | 9.5
.svelte | 114 | 41233 | 361.69
.ts | 86 | 3324 | 38.65
Total | 207 | 45171 |

**Comment by probablykasper** [comment]
.ts: 1610
.svelte: 2646 (1.6x)
